integration_tests/testsuite/test_monitoring.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python
+
+import json
+import logging
+import requests
+import time
+
+import test_util
+
+from google.cloud import monitoring as gcloud_monitoring
+
+
+def _test_monitoring(base_url):
```

Comment:
hmm, didn't think about it that way. I'll look into it, though I'm not sure I want to use any of python's built in testing frameworks.